examples/gnrc_border_router/README.md

```diff
@@ -60,3 +60,99 @@ for further help.
 [1] https://tools.ietf.org/html/rfc1055
 
 [2] https://github.com/contiki-os/contiki/blob/master/tools/tunslip.c
+
+
+# gnrc_border_router with ethos
+A new setup is possible on which only one serial interface is needed. This is done through the ethos (Ethernet Over Serial) driver that multiplexes serial data between ethernet packets and shell commands.
+
+## Requirements
+This functionality works only in Linux machines, Mac support will be added in the future (lack of `tap` interface), and Windows support should possible by setting up the needed `tap` interface with an external program (not tested).
+
+## Setup
+Take care about the modification of the `Makefile`, since by default SLIP is enabled.
+In order to select ethos as the driver, you should modify some lines as following:
+```make
+#CFLAGS += -DSLIP_UART=$(SLIP_UART)
+#CFLAGS += -DSLIP_BAUDRATE=$(SLIP_BAUDRATE)
+CFLAGS += '-DETHOS_UART=UART_DEV(0)' -DETHOS_BAUDRATE=115200 -DUSE_ETHOS_FOR_STDIO
+```
+Then deactivate the slip module and activate ethos
+```make
+# Include SLIP package for IP over Serial communication
+#USEMODULE += gnrc_slip
+USEMODULE += ethos gnrc_netdev2
+```
+This will allow to compile the BR firmware using ethos.
+Thus:
+```bash
+$ make clean all flash
+```
+should compile the example and flash it to your board.
+
+Afterwards, you need to compile the ethos tool instead of tunslip.
+To do this, in `dist/tools/ethos` you'll find the tool, just compile it with:
+```bash
+$ make
+```
+
+## Usage
+First, to have access to the nodes attached to the BR from your linux machine, it is necessary to add a `tap` interface as well as configure it to route the packets coming from your 6lo network.
+As an example, you can do the following:
+```bash
+$ sudo ip tuntap add tap0 mode tap user ${USER}
+```
+This will setup your `tap` interface, then:
+```bash
+$ sudo ip link set tap0 up
+$ sudo ip a a affe::1/48 dev tap0
+$ sudo ip r d affe::/48 dev tap0
+$ sudo ip r a affe::2 dev tap0
+$ sudo ip r a affe::/48 via affe::2 dev tap0
+```
+will configure the interface and add some routes in order to have access to the 6lo network from Linux.
+
+Finally, run ethos by choosing the right serial interface (`/dev/ttyUSB*` or `/dev/ttyACM*` according to your board):
+```bash
+$ sudo ./ethos tap0 /dev/ttyACM0
+```
+Then you can flash the BR example on your board, then add some addresses and a route:
+```
+> ifconfig 6 add affe::2/48
+> ifconfig 5 add affe::3/64
+> fibroute add :: via <link-local of tap> dev 6
+```
+
+Here we have choosen `affe` as the prefix. It should be disseminated to your other nodes on the radio range.
+
+Now, you should be able to ping your nodes using the global addres starting by your prefix, on our case `affe::`.
+
+# gnrc_border_router using automatic configuration (not merged yet)
+In \#4725 (https://github.com/RIOT-OS/RIOT/pull/4725) an effort to automate all the previous steps is done using 3 tools:
```

Comment:
You can have proper links in markdown doing this

```markdown
[\#4725](https://github.com/RIOT-OS/RIOT/pull/4725)
```

(and it's not too disruptive for local readers either)